nhaystack-rt/src/nhaystack/res/blacklist.txt

```diff
@@ -1,9 +1,9 @@
 actions
 axAnnotated
-axHistoryId
+n4HistoryId

```

Comment:
I need to review how this blacklist is used before approving this change and the same a few lines down.  If this is used elsewhere then we probably need to maintain the existing 'ax' tags.